### PR TITLE
ship conu in a container

### PR DESCRIPTION
**Pull request by TomasTomecek**

### CODE REVIEW

**Comment by psklenar** [comment]
## jenkins/superCI comment
* Status: failure, return code = __1__ 
* Result url: __[https://da.gd/uUXN](https://da.gd/uUXN)__
* Comment: __This is testing CI__

**Comment by TomasTomecek** [comment]
The CI failure is unrelated (it's poorly written test).

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+# this is a distribution mechanism of conu
+
+FROM registry.fedoraproject.org/fedora:26
+
+# the tests are being expected here
+VOLUME /tests
+WORKDIR /tests
+CMD ["pytest-3", "-v", "/tests"]
+
+RUN dnf install -y acl python3-pyxattr python3-pytest python3-docker python3-six docker-latest docker atomic && dnf clean all
```
Comment (jscotka) [inline Dockerfile]: why to install just `python3-pytest` and not `python2-pytest` it is not mandatory for conu to install pytest anyhow. It is test frametowrk (used by internal testsuite)

But it is good to have these test requirements somewhere mentioned.

**Comment by bkabrda** [comment]
So I think this PR is slightly confusing - as @jscotka mentioned above, testing is only one purpose of conu. I think it would make sense if this was shipped e.g. as Dockerfile.conu-pytest.

An additional question: is the `/tests` volume supposed to contain tests of a project that uses conu for testing? If so, then I think it'd be better to create a `/project` dir and allow copying both tests and actual code of upstream project inside - most projects need the local importable version of the code for tests to work. (If this was not the intention, then please explain the `/tests` directory purpose).

**Comment by TomasTomecek** [comment]
Very solid comments guys! I agree with you that adding testing packages in the container wasn't very good. I used the former container image to run tests in our internal CI.

I changed this PR to just install conu in a container image. PTAL

**Comment by bkabrda** [comment]
LGTM 👍

**Pull request merged**